modules/core/src/shaderlib/project/viewport-uniforms.js

```diff
@@ -181,16 +208,22 @@ function calculateViewportUniforms({
     project_uCameraPosition: cameraPos
   };
 
-  if (coordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS) {
-    const distanceScalesAtOrigin = viewport.getDistanceScales(coordinateOrigin);
+  if (shaderCoordinateSystem === COORDINATE_SYSTEM.METER_OFFSETS) {
```

Comment:
Nit: feels like a switch statement would be clearer?

Reply:
Next PR.